### DISCUSSION TITLE
How can I delete a rowset if I made a mistake when padding it?

### DISCUSSION THREAD

**Discussion by kenlin8827** [apache/doris#41457] [A - General / Q&A]
How can I delete a rowset if I made a mistake when padding it?

**Comment by boluor** [comment]
u could refer to [https://doris.apache.org/zh-CN/docs/3.0/sql-manual/sql-statements/Database-Administration-Statements/ADMIN-SET-REPLICA-STATUS?_highlight=set&_highlight=bad#example](url) , use set bad or other operation
